src/main/java/org/jfrog/hudson/pipeline/common/types/buildInfo/Env.java

```diff
@@ -34,10 +34,10 @@ public Env() {
      * Collect environment variables and system properties under with filter constrains
      */
     public void collectVariables(EnvVars env, Run build, TaskListener listener) {
-        EnvVars buildParameters = Utils.extractBuildParameters(build, listener);
-        if (buildParameters != null) {
-            env.putAll(buildParameters);
-        }
+        collectVariables(env).collectVariables(build, listener);
+    }
+
+    public Env collectVariables(EnvVars env) {
```

Comment:
Please add JavaDoc explaining that this method collects environment and system variables